### PR TITLE
Updated search field

### PR DESCRIPTION
**Pull request by CAnorbo**
The new search field with Algolia can search inside the content. Fix for: #1612

### CODE REVIEW

**Comment by CAnorbo** [comment]
@turbolay

**Review by yahiheb — CHANGES_REQUESTED** [review]
I get this error when trying to test this:

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find package '@vuepress/plugin-docsearch' imported from C:\Users\User\OneDrive\Bureau\WasabiDoc\docs\.vuepress\config.ts.3611753a.mjs
    at new NodeError (node:internal/errors:393:5)
    at packageResolve (node:internal/modules/esm/resolve:860:9)
    at moduleResolve (node:internal/modules/esm/resolve:909:20)
    at defaultResolve (node:internal/modules/esm/resolve:1124:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:841:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36)
```

**Comment by CAnorbo** [comment]
Try to run: npm install

**Comment by turbolay** [comment]
It works for me but I had several problems with it I'm trying to compile them

**Review by yahiheb — COMMENTED** [review]
> Try to run: npm install

Thanks, that worked.

I tested this a bit and it looks fine, let's wait for Clement's review.

**Comment by turbolay** [comment]
Sorry I took time I was busy:
- The bar is too small, it was bigger in the past
- There are many cases like this: 
CleanShot 2023-11-09 at 16 42 59@2x
 the link for this blank item is https://docs.wasabiwallet.io/using-wasabi/ColdWasabi.html#cold-wasabi-hardware-wallet-mode
- It is too permissive, it gives us irrelevant results. Better here show no result
CleanShot 2023-11-09 at 16 43 58@2x

**Comment by CAnorbo** [comment]
> Sorry I took time I was busy:
> 
> * The bar is too small, it was bigger in the past
> * There are many cases like this:
>   CleanShot 2023-11-09 at 16 42 59@2x
>   the link for this blank item is https://docs.wasabiwallet.io/using-wasabi/ColdWasabi.html#cold-wasabi-hardware-wallet-mode
> * It is too permissive, it gives us irrelevant results. Better here show no result
> 
> CleanShot 2023-11-09 at 16 43 58@2x

I made the search bar bigger, but I can't rewrite how is it working. I checked their website and even their search bar works the same.
There are a few options on their documentation but, not for these kinds of things.
https://docsearch.algolia.com/docs/api/#searchparameters
https://www.algolia.com/doc/api-reference/search-api-parameters/

File: docs/.vuepress/styles/index.scss [hunk 1]
```diff
@@ -84,11 +84,28 @@ html.dark {
 html:not(.dark){
 	--c-text-light: var(--c-text) !important;
 }
+.navbar .navbar-items-wrapper {
+	width: calc(100% - 250px) !important;
```
Comment (yahiheb) [inline docs/.vuepress/styles/index.scss:88]: image

This looks unnecessarily too big IMO, specially we cannot type in it.

I tested with `80% ` and it looks great.

```suggestion
	width: calc(80% - 250px) !important;
```

**Review by yahiheb — CHANGES_REQUESTED** [review]
> I made the search bar bigger, but I can't rewrite how is it working. I checked their website and even their search bar works the same.

The PR is good as is, this can be improved in the future of course

File: docs/.vuepress/styles/index.scss (hunk 1 above)
Comment (CAnorbo) [reply docs/.vuepress/styles/index.scss:88]: With this change, the search field won't be visible on some smaller desktop / tablet screens.

Comment (CAnorbo) [reply docs/.vuepress/styles/index.scss:88]: I added a different change that makes the search bar smaller on bigger screens. Maybe this solves the current problem.

Comment (yahiheb) [reply docs/.vuepress/styles/index.scss:88]: It is still the same for me.

Comment (turbolay) [reply docs/.vuepress/styles/index.scss:88]: On my screen it looks perfect now. On mobile it takes all the length and I think it's cool

Comment (CAnorbo) [reply docs/.vuepress/styles/index.scss:88]: > It is still the same for me.

What is your screen size?

Comment (turbolay) [reply docs/.vuepress/styles/index.scss:88]: 13.6", 3420x2080 
for mobile I use chrome "mobile view"

CleanShot 2023-11-13 at 02 55 35@2x

Comment (CAnorbo) [reply docs/.vuepress/styles/index.scss:88]: I meant Yahiheb. If you have 1590px or wider screen, you'll see the difference if you have smaller, you won't see any changes.

Comment (yahiheb) [reply docs/.vuepress/styles/index.scss:88]: 17-inch screen, 3840 x 2400

Comment (CAnorbo) [reply docs/.vuepress/styles/index.scss:88]: Try to empty cache or rerun the docs, you should see the same as Turbolay.

Comment (yahiheb) [reply docs/.vuepress/styles/index.scss:88]: It still the same for me.

**Review by yahiheb — APPROVED** [review]
The search field still looks too big for me, but I will merge this and I will ping you @CAnorbo  to improve this if needed.

**Pull request merged**